components/ButtonBar/src/MDCButtonBar.m

```diff
@@ -446,7 +446,7 @@ - (void)setButtonsTitleFont:(UIFont *)font forState:(UIControlState)state {
 
   for (NSUInteger i = 0; i < [_buttonViews count]; ++i) {
     UIView *viewObj = _buttonViews[i];
-    if ([viewObj isKindOfClass:[UIButton class]]) {
+    if ([viewObj isKindOfClass:[MDCButton class]]) {
       MDCButton *button = (MDCButton *)viewObj;
```

Comment:
I think in this case because we're not using any traits of MDCButton we can just cast to a UIButton instead. What do you think?

Reply:
`setTitleFont` looks specific to MDCButton, so this would have to remain.

Reply:
Cool - thanks for checking!